src/main/java/me/glasscrab/plentiful_harvest/Listeners/CropBreaks/CropBreakEvent.java

```diff
@@ -25,76 +25,94 @@ public CropBreakEvent(Manager manager) {
         this.manager = manager;
     }
 
+    /*  Function: When a crop breaks, run the function.
+        Parameters: BlockDropItemEvent e
+        Return: void */
```

Comment:
```suggestion
    /* 
        When a crop breaks, run the function.
        @param event The event being fired
        @returns void
    */
```

Reply:
Fixed all the comments and added some to the manager file